helix-view/src/view.rs

```diff
@@ -11,7 +11,7 @@ type Jump = (DocumentId, Selection);
 
 #[derive(Debug, Clone)]
 pub struct JumpList {
-    jumps: Vec<Jump>,
+    pub jumps: Vec<Jump>,
```

Comment:
How about rather than making `jumps` `pub`, we add a `pub fn remove(&mut self, view_id: ViewId)`?

Reply:
Since `Vec<Jump>` == `Vec<(DocumentId, Selection)>`, the actual signature is `pub fn remove(&mut self, doc_id: &DocumentId)` -- is that acceptable?